_output/golang-versions

```diff
@@ -0,0 +1,3 @@
+1.20
```

Comment:
We shoudn't push these files generated in _output

Reply:
oh, nice gotcha.

do you know why they are not part of `.gitignore`?